### ISSUE TITLE
Queued experiments fail, can't see logs to understand what's up

### ISSUE THREAD

**Issue by shcheklein** [iterative/vscode-dvc#4332]
https://github.com/iterative/vscode-dvc/assets/3659196/d2e544bb-0de3-440b-9769-2609b996093c

Single experiment runs fine even within a queue. Might be related to some resource allocation when it tries to do 2 of them, but I can't say what's going on.

It's related to me researching user complaining:

> But if i run the same queue again..No metrics are getting displayed in Studio
> Also these logged experiments are not getting deleted

**Comment by mattseddon** [comment]
Depends on https://github.com/iterative/dvc/issues/9425

**Comment by shcheklein** [comment]
cc @dberenbaum

**Comment by dberenbaum** [comment]
@shcheklein Does `dvc queue logs motor-abac` work?

**Comment by mjunker** [comment]
I run into the same issue. My impression is that this happens mainly when running lots of experiments (100 or more). I create experiments using the cli like this:

`dvc exp run -S <several params with ranges> --queue --temp -f -n <name>
dvc queue start -j 8`

Also I do not get any output from dvc queue logs <exp-name>. When I apply the failed experiment to workspace it runs without any problem.

**Comment by shcheklein** [comment]
Making it `p1` (even though it's blocked) since it comes my very often (cc @dberenbaum - let me know what you think). For the record we had also a new ticket #4524 , also I think I saw it today on Discord.

**Comment by shcheklein** [comment]
@dberenbaum I think eventually we try to cover both. If I understand correctly the `Queued experiments fail to generate logs if error happened before stage runs.`. Those errors could happen if there is no data, or someone forgot to commit params.yaml, etc, etc. If there is no easy way to see those errors and then restart those experiments it becomes a problem during onboarding. 

The first item will be fixed automatically if the first one is fixed, right?

What is your take? What is the complexity and scope of both on the DVC side?

**Comment by dberenbaum** [comment]
> The first item will be fixed automatically if the first one is fixed, right?

What did you mean here? I don't know that either one will fix the other. We are discussing in the tickets above what the options are to solve each and what level of effort it takes.

**Comment by RawthiL** [comment]
This is old, but I'm having the same problem with dvc `3.59.2`, both on Linux and OSX. All changes comited and pushed before starting any queue or experiment. 

I created a **very basic** experiment, that only logs random stuff (just like the example) and I added a single parameter to set the randomness seed. I can run the experiment completely fine with:

`dvc exp run --set-param general.seed=99`

Seed is correctly set and everything looks normal. Now, when I want to run a simple queue, I do:

`dvc exp run --queue --set-param "general.seed=8,16,24" --name "grid-test"`

the result of the status seems fine:
```
➜ dvc queue status            
Task     Name           Created       Status
a5c6063  grid-test-1  11:02 AM      Queued
d27ee3f  grid-test-2  11:02 AM      Queued
75a976e  grid-test-3  11:02 AM      Queued
```

then I run the queue:

`dvc queue start -vvv`

and this is the only log I get:

```
2025-05-26 11:14:44,697 DEBUG: v3.59.2 (pip), CPython 3.12.3 on Linux-6.11.0-26-generic-x86_64-with-glibc2.39
2025-05-26 11:14:44,697 DEBUG: command: /home/rawthil/.cache/pypoetry/virtualenvs/******-021--Ev3-py3.12/bin/dvc queue start -vvv
2025-05-26 11:14:44,697 TRACE: Namespace(quiet=0, verbose=3, cprofile=False, cprofile_dump=None, yappi=False, yappi_separate_threads=False, viztracer=False, viztracer_depth=None, viztracer_async=False, pdb=False, instrument=False, instrument_open=False, show_stack=False, cd='.', cmd='start', jobs=1, func=<class 'dvc.commands.queue.start.CmdQueueStart'>, parser=DvcParser(prog='dvc', usage=None, description='Data Version Control', formatter_class=<class 'dvc.cli.formatter.RawTextHelpFormatter'>, conflict_handler='error', add_help=False))
2025-05-26 11:14:44,856 DEBUG: Spawning 1 exp queue workers
2025-05-26 11:14:45,865 DEBUG: Worker status: {}
2025-05-26 11:14:45,872 DEBUG: Spawning exp queue worker
2025-05-26 11:14:45,872 DEBUG: start a new worker: dvc-exp-worker-1, node: dvc-exp-d11349-1@localhost
Started '1' new experiments task queue worker.
2025-05-26 11:14:45,876 DEBUG: Analytics is enabled.
2025-05-26 11:14:45,876 TRACE: Saving analytics report to /tmp/tmph4h3xae6
2025-05-26 11:14:45,894 DEBUG: Trying to spawn ['daemon', 'analytics', '/tmp/tmph4h3xae6', '-vv']
2025-05-26 11:14:45,897 DEBUG: Spawned ['daemon', 'analytics', '/tmp/tmph4h3xae6', '-vv'] with pid 272195
2025-05-26 11:14:45,898 TRACE: Process 272182 exiting with 0
```

and then all queue fails:

```
➜ dvc queue status          
Task     Name           Created       Status
a5c6063  grid-test-1  11:02 AM      Failed
d27ee3f  grid-test-2  11:02 AM      Failed
75a976e  grid-test-3  11:02 AM      Failed
```

The VSCode extension only says "experiment failed" and there are no logs:

```
➜ dvc queue logs grid-test-3
ERROR: No output logs found for experiment 'grid-test-3'
```

Any help with this?
